flink-connectors/flink-orc/src/main/java/org/apache/flink/orc/OrcTableSource.java

```diff
@@ -213,7 +213,8 @@ public boolean isFilterPushedDown() {
 
 	@Override
 	public String explainSource() {
-		return "OrcFile[path=" + path + ", schema=" + orcSchema + ", filter=" + predicateString() + "]";
+		return "OrcFile[path=" + path + ", schema=" + orcSchema + ", filter=" + predicateString()
```

Comment:
had to change this override since in fact the OrcTableSource does not explain the pushed down predicate.